documentation/guide-how-to-create-a-devon4quarkus-app.asciidoc

```diff
@@ -0,0 +1,224 @@
+
+To load a new Quarkus application from command line with the Quarkus Maven plugin run the following command:
+
+.Operating system
+. Linux and MacOs +
+`mvn io.quarkus:quarkus-maven-plugin:2.0.2.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=getting-started \
+    -DclassName="org.acme.getting.started.GreetingResource" \
+    -Dpath="/hello"`
+
+. Windows +
+    `mvn io.quarkus:quarkus-maven-plugin:2.0.2.Final:create "-DprojectGroupId=org.acme" "-DprojectArtifactId=getting-started" "-DclassName=org.acme.getting.started.GreetingResource" "-Dpath=/hello"`
+ 
+
+it generates the following in the getting-started folder
+
+* the Maven structure
+* an org.acme.getting.started.GreetingResource resource exposed on /hello
+* an associated unit test
+* a landing page that is accessible on http://localhost:8080 after starting the application
+* example Dockerfile files for both native and JVM modes in src/main/docker
+* the application configuration file
+
+Once generarted change your directory with `cd getting-started` and take a look at the *pom.xml*. You will find the import of the Quarkus BOM, allowing you to omit the version on the different Quarkus dependencies. In addition, you can see the *quarkus-maven-plugin* responsible for packaging of the application and also providing the development mode.
+
+[source, xml]
+----
+  <properties>
+<artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+    <quarkus.platform.version>2.0.2.Final</quarkus.platform.version>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>${quarkus.platform.group-id}</groupId>
+        <artifactId>${quarkus.platform.artifact-id}</artifactId>
+        <version>${quarkus.platform.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+<build>
+    <plugins>
+        <plugin>
+            <groupId>${quarkus.platform.group-id}</groupId>
+            <artifactId>quarkus-maven-plugin</artifactId>
+            <version>${quarkus.platform.version}</version>
+            <extensions>true</extensions>
+            <executions>
+                <execution>
+                    <goals>
+                        <goal>build</goal>
+                        <goal>generate-code</goal>
+                        <goal>generate-code-tests</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+    </plugins>
+</build>
+----
+
+==== Extension management
+Extensions added by creating a project from https://code.quarkus.io or with the parameter `-Dextensions` in your mvn create command and can be found after that in the pom.xml. Now we are just using *RESTEasy*. 
```

Comment:
This is not quite correct. Extensions for testing are also added. And implicitly many other dependencies are added because the RESTEasy extension needs them

Reply:
You are right the wording is bad, have to rewrite the sentence.